xgcm/test/test_grid.py

```diff
@@ -782,3 +782,26 @@ def test_boundary_kwarg_same_as_grid_constructor_kwarg():
     actual2 = grid2.interp(ds.data_g, ("X", "Y"))
 
     xr.testing.assert_identical(actual1, actual2)
+
+
+@pytest.mark.parametrize(
+    "metric_axes,metric_name",
+    [
+        ("X", "dx_t"),
+        ("Y", "dy_ne"),
```

Comment:
You should be able to add `("['Y', 'X']", "dy_n"), and this should still work. If that does not work, just ignore the comment.

Reply:
Unfortunately it returned this error: `Fail: KeyError: 'Metric axes ["[\'Y\', \'X\']"] not compatible with grid axes (\'X\', \'Y\', \'Z\')'`

Reply:
Sorry that was my fault. It should be `['Y', 'X']`, not `"['Y', 'X']"`